internal/interfaces/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package interfaces contains global interfaces for objects, etc.
+package interfaces
```

Comment:
We don't need an additional package for this. If they are public, then `pkg/http` is a better fit.